src/AssemblySharedInfoGenerator/AssemblySharedInfo.tt

```diff
@@ -68,7 +66,7 @@ using System.Runtime.InteropServices;
 [assembly: AssemblyFileVersion("<#= this.MajorVersion #>.<#= this.MinorVersion #>.<#= this.BuildNumber #>.<#= this.RevisionNumber #>")]
 <#+
 int MajorVersion = 3;
-int MinorVersion = 0;
+int MinorVersion = 1;
```

Comment:
Can you update the year in the `RevisionNumber` formula as well to use 2024?

Reply:
Goto 2024 will give negative numbers but we can do 2023 now and then try to remember to bump it again when we get to 3.2.

Reply:
Sure, thanks!